frontend/public/components/dashboard/project-dashboard/resource-quota-card.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardLink from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardLink';
+import { getQuotaResourceTypes, QuotaGaugeCharts, QuotaScopesInline } from '../../resource-quota';
+import { ResourceLink, resourcePath } from '../../utils';
+import { ResourceQuotaModel } from '../../../models';
+
+export const ResourceQuotaCard = ({ resourceQuota }) => {
+  const resourceTypes = getQuotaResourceTypes(resourceQuota);
+  const scopes = _.get(resourceQuota, 'spec.scopes');
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <div>
+          <ResourceLink
+            kind={ResourceQuotaModel.kind}
+            name={resourceQuota.metadata.name}
+            className="co-resource-item--truncate"
+            namespace={resourceQuota.metadata.namespace}
+            inline="true"
+            title={resourceQuota.metadata.name}
+            linkTo={false}
+          />
+          {scopes && (
+            <QuotaScopesInline className="co-resource-quota-dashboard-scopes" scopes={scopes} />
+          )}
+        </div>
+        <DashboardCardLink
+          to={resourcePath(
+            ResourceQuotaModel.kind,
+            resourceQuota.metadata.name,
+            resourceQuota.metadata.namespace,
+          )}
+        >
+          See more
```

Comment:
```suggestion
          View details
```

This would align with the rest of the dashboard's wording (assuming this link takes the user to the Details page of the ResourceQuota).